backend/dataall/api/Objects/Environment/queries.py

```diff
@@ -11,6 +11,12 @@
     test_scope='Environment',
 )
 
+getCreatePivotRole = gql.QueryField(
```

Comment:
I commented it somewhere below, I feel like we don't need to expose an additional query for it